addons/resource.language.en_gb/resources/strings.po

```diff
@@ -5285,7 +5285,12 @@ msgctxt "#12373"
 msgid "Settings & file manager"
 msgstr ""
 
-#empty strings from id 12374 to 12375
+#empty strings with id 12374
+
+#: xbmc\video\dialogs\GUIDialogVideoSettings.cpp
+msgctxt "#12375"
+msgid "The selected values will be saved as default for all media, and also affect playback. Are you sure?"
```

Comment:
what about `... and affect how they are played back`? I'm no native speaker, so I don't know if your short form is clear enough. @jjd-uk - mind adding some paint?